lib/simctl.ts

```diff
@@ -117,7 +117,9 @@ export class Simctl implements ISimctl {
 	}
 
 	private simctlExec(command: string, args: string[], opts?: any): any {
-		let fullCommand = (["xcrun", "simctl", command].concat(args)).join(" ");
+		let escapedArgs = args.map((arg) => `"${arg}"`);
```

Comment:
I do believe this code will fail if `arg` contains a quotation mark.

I'd suggest you use `childProcess.spawnSync` instead of `childProcess.execSync` so that `nodejs` takes care of escaping for you.

Bear in mind, however, that `spawnSync` has a slightly different return result so in order to keep the current behavior you'd have to change the return statement to something like
`return childProcess.spawnSync("xcrun", ["simctl", command].concat(args), opts).stdout.toString().trim();`